### ISSUE TITLE
how to set temporary home directory

### ISSUE THREAD

**Issue by ahmedabbas811** [nservant/HiC-Pro#650]
Hello,

I am trying to run HiC-Pro on a cluster (slurm environment)

I type:
`HiC-Pro -i ./H3K27ac_rep2 -o ./Out_dir -c ./config.txt`

It gives the following error:
```
Starting HiC-Pro 3.1.0 via Singularity
No temporary home directory set. Using /home2 for your home directory.
Error - Hi-C config file '/home2/s206442/config.txt' not found
```

How to change the temporary home directory to the current directory?

Thanks

**Comment by nservant** [comment]
I do not think that this message is sent by HiCPro itself ... I think this is more related to `singularity` and its configuration. I would advice to explore the singularity options ...
